cmd/rpc.go

```diff
@@ -69,6 +71,31 @@ func InitClient(cmd *cobra.Command, _ []string) error {
 	return nil
 }
 
+func getStorePath(cmd *cobra.Command) (string, error) {
+	// if node store flag is set, use it
+	if cmd.Flag(nodeStoreFlag).Changed {
+		return cmd.Flag(nodeStoreFlag).Value.String(), nil
+	}
+
+	// try to detect a running node by checking for a lock file
+	nodeTypes := []nodemod.Type{nodemod.Bridge, nodemod.Full, nodemod.Light}
+	defaultNetwork := []p2p.Network{p2p.Mainnet, p2p.Mocha, p2p.Arabica, p2p.Private}
+	for _, t := range nodeTypes {
+		for _, n := range defaultNetwork {
+			path, err := DefaultNodeStorePath(t.String(), n.String())
+			if err != nil {
+				return "", err
+			}
+
+			if nodebuilder.IsOpened(path) {
+				return path, nil
+			}
+		}
+	}
```

Comment:
Let's also move it next to `IsOpened` as `DiscoverOpened` returning the path and the error if not found.

Reply:
moved, had to move DefaultNodeStorePath as well